Dockerfile

```diff
@@ -61,12 +61,6 @@ ENV JAVA_HOME_8_X64=/usr/lib/jvm/java-8-openjdk-amd64 \
     JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 \
     JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8
 
-# Install Docker
-RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - > /dev/null \
```

Comment:
is there a reason you're removing this? remember this image is shared.

I would leave it there and just not mount the socket when you're using the agent

Reply:
Added back.  Must have misunderstood previous discussions about the security considerations around this.  Make sense to leave however given it's shared.